CONTRIBUTING.md

```diff
@@ -183,3 +183,9 @@ ampersand (`&`), e.g., `feature_flights="flight1&flight2&flight3"`.
 
 In the dashboard test environment (using `yarn start`) you have a dropdown to
 select which flights should be active.
+
+### Code approvals
+
+Once you have made your code changes locally, committed them and verified them, you can send a pull request (in short form written as PR) to [responsible-ai-toolbox](https://github.com/microsoft/responsible-ai-toolbox). For more information on how to create a pull request, please see [Proposing changes to your work with pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests).
```

Comment:
Thanks for adding this. How do you feel about letting people know that they can't create PRs from their own forks (due to the Azure blob permissions issue in the CI build)?

Reply:
I feel like that's more of a bug that we need to fix.  External users won't actually be able to push to the branches on our github, only to their forks.  Hence, this issue basically blocks them from contributing in the normal way (but ofcourse we can always force merge, as we have already been doing for some external contributions).